balancer/balancer.go

```diff
@@ -335,10 +335,12 @@ func (b *BalancerImpl) GetBestNode(ctx context.Context, redirectPrefixes []strin
 	return fallbackNode(fallbackAddr, fallbackPrefix, playbackID, redirectPrefixes[0], err)
 }
 
+// `playbackID`s matching the pattern `....-....-.....-....`
+var regexpPlaybackID = regexp.MustCompile(`^(?:\w{4}-){3}\w{4}$`)
```

Comment:
Why do we need playback id to match this patter? 🤔

Reply:
It's not actually a playback id if it's in this format! It's a stream key, and we match this to send them directly to a Mist endpoint instead of doing playback load balancing because it means they're streaming in 
```suggestion
var regexpStreamKey = regexp.MustCompile(`^(?:\w{4}-){3}\w{4}$`)
```